### ISSUE TITLE
Consider swapping Twilight and Fractal

### ISSUE THREAD

**Issue by luminaStuffphoton** [Arch-Shaman/ZK-Futurewars-Mod#555]
Twilight is a skirm with slow. A long range riotskirm with slow. Disgusting. What was I on to ever come up with this >w<
Fractal is weird. It's an anti heavy in a factory that already has drone spam and domi. Original Fencer was cooler
Proposal: Remove slow from Twilight and make it's missiles green. Give slow to Fractal and make it's missiles purple ^w^
Probably also give twilight back it's 3-shot burst each missile would only have 1/3rd the dps. it looks weird to see 3 rockets on the model but only 1 shot.

http://zero-k.info/Battles/Detail/1711377 may be of relevance

**Comment by LeojEspino** [comment]
Fencer with

**Comment by LeojEspino** [comment]
NGAAH I NEED TO STOP DOING THAT

**Comment by LeojEspino** [comment]
Also Fencer with slow sounds super duper op and oppressive :(

**Comment by IdkSomethingTemporary** [comment]
> NGAAH I NEED TO STOP DOING THAT

Um- is everything okay?

> Also Fencer with slow sounds super duper op and oppressive :(

Stat-wise OP or Mechanically OP, or in simpler terms, is there some cost value between 0 and infinity which you can assign to the unit for it to be balanced? Fencer is a riot-skirm hybrid and it is already hard to push into in vanilla. slow wouldn't really change that. It's counter both in vanilla and FW is arty and riot-arty, and slow wouldn't really change that. Slow would also push it more into the fire support role by making it want to be combo'd with dps units that can make use of the slow. 
Twilight on the other hand is a unit that you want to counter by moving your army into it. slow prevents that. Not sure if you've ever played vs vanila Moderator-placeholder but twilight is just the even more cancerous version of that with it's arty-skirm range compared to moderator's assault range
Of course, fencer would need a range nerf down to fire support range and not arty-skirm range

**Comment by LeojEspino** [comment]
I have never seen Twilight as more cancerous Moderator, maybe I didn't see enough players using it against my tanks to make me think its actually too annoying, also I guess Fencer and Fractal plays pretty much the same, just that the slow addition will actually make it more supporting. Also for that twilight's 1 missile shot but 3 missiles on the model I can probably remodel Twilight if I ever do to make it look more appropriate, though I never remodeled a spider before.

**Comment by IdkSomethingTemporary** [comment]
I'll give fractal slow and remove the heat part. Twilight could probably stay the same on second though, I'll make it less janky

**Comment by Arch-Shaman** [comment]
> I have never seen Twilight as more cancerous Moderator, maybe I didn't see enough players using it against my tanks to make me think its actually too annoying, also I guess Fencer and Fractal plays pretty much the same, just that the slow addition will actually make it more supporting. Also for that twilight's 1 missile shot but 3 missiles on the model I can probably remodel Twilight if I ever do to make it look more appropriate, though I never remodeled a spider before.

If you want to try go ahead. I'd prefer to see hover remakes though (especially pike and halberd). Speaking of pike, we should probably rework it into a support skirmisher at some point? (This is beyond the scope of this ticket though)

**Comment by IdkSomethingTemporary** [comment]
Seconded, it's a lot easier to make Twilight 3-burst but with 1/3rd the damage per missile

**Comment by Arch-Shaman** [comment]
Sounds good.
